fast-list.js

```diff
@@ -96,8 +93,8 @@ FastList.prototype = {
    *
    * @private
    */
-  renderPhase1: function() {
-    debug('render phase 1');
+  setupPhase1: function() {
```

Comment:
'setup' is a nicer word as it's clearer it is a once time thing and not associated with normally scroll rendering.